Big-Spender/readme.md

```diff
@@ -94,6 +112,7 @@ CREATE YOUR QUERY HERE
 ```sql
 INSERT YOUR QUERY HERE
 ```
+=> select sum (amount), supplier_id from suppliers join spends on suppliers.id = spends.supplier_id GROUP BY supplier_id;
```

Comment:
nice job! this looks good. what about using an alias for the sum (amount)? make it a bit cleaner :)